chronograf.go

```diff
@@ -762,6 +758,40 @@ type ConfigStore interface {
 	Update(context.Context, *Config) error
 }
 
+// OrganizationConfig is the organization config for parameters that can
+// be set via API, with different sections, such as LogViewer
+type OrganizationConfig struct {
+	OrganizationID string          `json:"organization"`
```

Comment:
is this just for consistency with other things that use `organization` as `organizationID`?

Reply:
seemed easier to think about since it's an id, and `organization` is ambiguous as a var name as to whether it's an organization name or an organization id

Reply:
yeah I totally agree. seems a little weird to me though that the name is `OrganizationID` on the struct and `organization` on the JSON.

Reply:
yeah.. it should've been `organizationID` on the JSON, but it's too late for that now. but i suppose i'd prefer to reason about it effectively on the code side, and accept that the client json calls it something else. thoughts?